index.js

```diff
@@ -18,7 +20,17 @@ github.authenticate({
     token: process.env.FORKED_TOKEN,
 })
 
-const packageJson = shell.cat('package.json')
+const args = process.argv.slice(2)
+if (args.indexOf('-h') !== -1 || args.indexOf('--help') !== -1) {
+  console.log(`  Easy-peasy usage:
+    1. Navigate to directory of dependency you want to fork.
+    2. run \`fork\`
+
+  Lemon-squeezy usage:
+    1. From your project’s root directory, run \`fork dependencyName\`.`)
```

Comment:
Got it. That certainly makes things much simpler. I agree with you that magical is not a very good idea for this particular feature. Thanks for elaborating on this!